modules/core/src/main/java/org/apache/ignite/client/ClientCluster.java

```diff
@@ -41,6 +42,19 @@ public interface ClientCluster extends ClientClusterGroup {
      */
     public void state(ClusterState newState) throws ClientException;
 
+    /**
+     * Changes current cluster state to given {@code newState} cluster state.
+     * <p>
+     * <b>NOTE:</b>
+     * Deactivation clears in-memory caches (without persistence) including the system caches.
+     *
+     * @param newState New cluster state.
+     * @param forceDeactivation If {@code true}, cluster deactivation will be forced.
+     * @throws ClientException If change state operation failed.
+     */
+    @IgniteExperimental
+    public void state(ClusterState newState, boolean forceDeactivation) throws ClientException;
```

Comment:
Corresponding method for a node implemented in private API - IgniteClusterEx.
Can we implement this into private thin client methods, also?